Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM progrium/busybox
+FROM golang:1.5.1-alpine
```

Comment:
Can we use 1.5.3 instead of 1.5.1.  1.5.3 contains a security fix - https://groups.google.com/d/topic/golang-announce/MEATuOi_ei4/discussion.